charts/brokencrystals/templates/deployment.yaml

```diff
@@ -27,7 +27,7 @@ spec:
             - "proxy"
             - "repeater"
             - "mailcatcher"
-            - "brokencrystals.local"
```

Comment:
Do we need this?

Reply:
Yes. For some reason, Bright doesn't want to scan first level domains